src/main/java/ExceptionHandling/InvalidCommandException.java

```diff
@@ -0,0 +1,6 @@
+package ExceptionHandling;
+
+
+public class InvalidCommandException extends Exception{
+
+}
```

Comment:
Empty Class. Please add at least a constructor with valid instantiations/assignments, so if this error is thrown, the description shown is correct. This has been observed in multiple files.

Else please add a comment explaining when would this error be thrown and why is it empty and nothing initialised.